package.json

```diff
@@ -32,7 +32,7 @@
     "@avcs/autosuggest": "^1.7.5",
     "@floip/expression-evaluator": "^2.1.0",
     "@floip/expression-parser": "^1.11.3",
-    "@floip/flow-runner": "^1.0.12-rc4",
+    "@floip/flow-runner": "https://github.com/FLOIP/flow-runner.git#feature/CORE-538/CORE-571/move-local-state-to-store",
```

Comment:
- [] publish flow-runner to nmp after we merge it, then consume it here

Reply:
Made a note in https://github.com/FLOIP/flow-builder/pull/472, as it might be a bit early for releasing the package.